packages/fiori/src/IllustratedMessage.ts

```diff
@@ -0,0 +1,525 @@
+	* <li><code>SimpleNotFoundMagnifier</code></li>
+	* <li><code>SimpleReload</code></li>
+	* <li><code>SimpleTask</code></li>
+	* <li><code>SleepingBell</code></li>
+	* <li><code>SortColumn</code></li>
+	* <li><code>SuccessBalloon</code></li>
+	* <li><code>SuccessCheckMark</code></li>
+	* <li><code>SuccessHighFive</code></li>
+	* <li><code>SuccessScreen</code></li>
+	* <li><code>Tent</code></li>
+	* <li><code>UnableToLoad</code></li>
+	* <li><code>UnableToLoadImage</code></li>
+	* <li><code>UnableToUpload</code></li>
+	* <li><code>UploadToCloud</code></li>
+	* <li><code>UploadCollection</code></li>
+	* <li><code>TntChartArea</code></li>
+	* <li><code>TntChartArea2</code></li>
+	* <li><code>TntChartBar</code></li>
+	* <li><code>TntChartBPMNFlow</code></li>
+	* <li><code>TntChartBullet</code></li>
+	* <li><code>TntChartDoughnut</code></li>
+	* <li><code>TntChartFlow</code></li>
+	* <li><code>TntChartGantt</code></li>
+	* <li><code>TntChartOrg</code></li>
+	* <li><code>TntChartPie</code></li>
+	* <li><code>TntCodePlaceholder</code></li>
+	* <li><code>TntCompany</code></li>
+	* <li><code>TntComponents</code></li>
+	* <li><code>TntExternalLink</code></li>
+	* <li><code>TntFaceID</code></li>
+	* <li><code>TntFingerprint</code></li>
+	* <li><code>TntLock</code></li>
+	* <li><code>TntMission</code></li>
+	* <li><code>TntNoApplications</code></li>
+	* <li><code>TntNoFlows</code></li>
+	* <li><code>TntNoUsers</code></li>
+	* <li><code>TntRadar</code></li>
+	* <li><code>TntSecrets</code></li>
+	* <li><code>TntServices</code></li>
+	* <li><code>TntSessionExpired</code></li>
+	* <li><code>TntSessionExpiring</code></li>
+	* <li><code>TntSuccess</code></li>
+	* <li><code>TntSuccessfulAuth</code></li>
+	* <li><code>TntSystems</code></li>
+	* <li><code>TntTeams</code></li>
+	* <li><code>TntTools</code></li>
+	* <li><code>TntUnableToLoad</code></li>
+	* <li><code>TntUnlock</code></li>
+	* <li><code>TntUnsuccessfulAuth</code></li>
+	* <li><code>TntUser2</code></li>
+	* </ul>
+	* <br><br>
+	* <b>Note:</b> By default the <code>BeforeSearch</code> illustration is loaded.
+	* <br>
+	* When using an illustration type, other than the default, it should be loaded in addition:
+	* <br>
+	* <code>import "@ui5/webcomponents-fiori/dist/illustrations/NoData.js";</code>
+	* <br><br>
+	* <b>Note:</b> TNT illustrations cointain <code>Tnt</code> prefix in their name.
+	* You can import them removing the <code>Tnt</code> prefix like this:
+	* <br>
+	* <code>import "@ui5/webcomponents-fiori/dist/illustrations/tnt/SessionExpired.js";</code>
+	* @type {sap.ui.webc.fiori.types.IllustrationMessageType}
+	* @defaultvalue "BeforeSearch"
+	* @name sap.ui.webc.fiori.IllustratedMessage.prototype.name
+	* @public
+	*/
+	@property({ type: IllustrationMessageType, defaultValue: IllustrationMessageType.BeforeSearch })
+	name!: IllustrationMessageType;
+
+	/**
+	* Illustration breakpoint variant for the <code>Spot</code> size.
+	*
+	* @private
+	* @type {String}
+	* @since 1.9.0
+	*/
```

Comment:
type: String is by default for the property decorator and we can skip it, as done for several properties above.